tests/apps/pubsub-subscriber/app.go

```diff
@@ -91,6 +96,15 @@ func configureSubscribeHandler(w http.ResponseWriter, r *http.Request) {
 func subscribeHandler(w http.ResponseWriter, r *http.Request) {
 	log.Printf("aHandler is called %s\n", r.URL)
 
+	if respondWithError {
+		// do not store received messages, respond with error
+		w.WriteHeader(http.StatusInternalServerError)
+		json.NewEncoder(w).Encode(appResponse{
+			Message: "retry later",
+			Status:  "RETRY",
```

Comment:
`RETRY` messages will be retried. 
https://github.com/dapr/dapr/blob/master/pkg/runtime/runtime.go#L1107